app/models/user.rb

```diff
@@ -51,8 +55,11 @@ class User < ActiveRecord::Base
   validate(:password_reset_cant_be_email_if_no_email)
   validate(:no_duplicate_assignments)
   validate(:must_have_assignments_if_not_admin)
-  validate(:phone_should_be_unique)
-  validates :password, format: { with: /(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])/, if: :require_password?, message: :invalid_password }
+  validate(:phone_should_be_unique, unless: :skip_validations_with_queries?)
+  validates :password, format: { with: /(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])/,
+                                 if: :require_password?,
+                                 unless: :skip_validations_with_queries?,
```

Comment:
You are right! It started being a flag for skipping validations with queries, but extended to be a global flag for all unnecessary validations on batch creation. Will change here. Thanks!